mlfinlab/labeling/labeling.py

```diff
@@ -67,11 +67,16 @@ def add_vertical_barrier(t_events, close, num_days=1):
 
     :param t_events: (series) series of events (symmetric CUSUM filter)
     :param close: (series) close prices
-    :param num_days: (int) maximum number of days a trade can be active
+    :param num_days: (int) number of days to add for vertical barrier
+    :param num_hours: (int) number of hours to add for vertical barrier
+    :param num_minutes: (int) number of minutes to add for vertical barrier
+    :param num_seconds: (int) number of seconds to add for vertical barrier
     :return: (series) timestamps of vertical barriers
     """
+    timedelta = pd.Timedelta(
```

Comment:
Do we have a unit test that checks different combinations of timestamps? That they work as expected?